emoji/pom.xml

```diff
@@ -23,6 +23,7 @@
             <groupId>org.openjfx</groupId>
             <artifactId>javafx-controls</artifactId>
             <version>${javafx.version}</version>
+            <scope>provided</scope>
```

Comment:
we need to document this behavior. 
probably the sample will simply work, because it will obviously depend on openjfx itself, but it's good to make it explicit in the readme

Reply:
Readme says:
> This module is to be used as a dependency in your JavaFX application.

A JavaFX application must have JavaFX dependencies, so I don't think we need to explicitly say that emoji doesn't bring any extra dependency, particularly any JavaFX dependency?